resources/options_lookup.tsv

```diff
@@ -9693,6 +9693,8 @@ HVAC Cooling Type	Evaporative or Swamp Cooler
 HVAC Cooling Type	Non-Ducted Heat Pump
 HVAC Cooling Type	None
 HVAC Cooling Type	Room AC
+HVAC Detailed Performance Data	CCHP Ducted	ResStockArguments	hvac_perf_data_capacity_type=Normalized capacity fractions	hvac_perf_data_heating_outdoor_temperatures=67.0, 47.0, 17.0, 5.0, -15.0	hvac_perf_data_heating_min_speed_capacities=0.423, 0.308, 0.353, 0.371, 0.331	hvac_perf_data_heating_max_speed_capacities=1.387, 1.028, 0.766, 0.688, 0.507	hvac_perf_data_heating_min_speed_cops=5.150, 4.390, 2.550, 2.050, 1.660	hvac_perf_data_heating_max_speed_cops=3.684, 3.370, 2.330, 1.980, 1.490	hvac_perf_data_cooling_outdoor_temperatures=125.0, 95.0, 82.0, 70.0	hvac_perf_data_cooling_min_speed_capacities=0.267, 0.333, 0.368, 0.391	hvac_perf_data_cooling_max_speed_capacities=0.847, 1.017, 1.090, 1.148	hvac_perf_data_cooling_min_speed_cops=1.918, 3.870, 5.860, 7.126	hvac_perf_data_cooling_max_speed_cops=2.017, 3.220, 4.140, 4.900
```

Comment:
I feel like we should spell out "Cold Climate Heat Pump" in these new option names.